<ui version="4.0" >
 <class>form_chatwidget</class>
 <widget class="QWidget" name="form_chatwidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>User Chat</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTextBrowser" name="chat" >
      <property name="openExternalLinks" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayout" >
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="txtBold" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="minimumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text" >
            <string>B</string>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff/>
            </iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
           <property name="autoRaise" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="txtUnder" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="minimumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text" >
            <string>U</string>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff/>
            </iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="txtItalic" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="minimumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text" >
            <string>I</string>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff/>
            </iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="txtColor" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="minimumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text" >
            <string>C</string>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff/>
            </iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="txtFont" >
           <property name="minimumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text" >
            <string>F</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="message" />
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="send" >
           <property name="text" >
            <string>Senden</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../temp/client.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>txtItalic</sender>
   <signal>clicked(bool)</signal>
   <receiver>message</receiver>
   <slot>setFontItalic(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>644</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel" >
     <x>348</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtUnder</sender>
   <signal>clicked(bool)</signal>
   <receiver>message</receiver>
   <slot>setFontUnderline(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>614</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel" >
     <x>348</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
